src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -27,8 +25,395 @@
     content: "·";
 }
 
-// Theming
-build-themes-Enactors(themes);
+// General theme styling that applies to html elements and infusion components.
+// !important is used for all styles to ensure that Preferences Framework's enactors are able to override the pages default styles.
+[class*="fl-theme"]:not(.fl-theme-prefsEditor-default) {
```

Comment:
What I was suggesting with [FLUID-6552](https://issues.fluidproject.org/browse/FLUID-6552) was to allow for the composition of enactor styles into this single file or output as individual CSS files. In that sense I'm not sure moving this styling form the Themes.style buys much. Unless you think we'll have more cases where we want to include the variables without related styling.

Reply:
My thinking here was that if we do split enactors styling into individual stylesheets we'd need the ability to import the variables into individual enactor stylesheets as well as a combined stylesheet like the current `Enactors.css`. I think it makes sense to address those structural changes in work on [FLUID-6552](https://issues.fluidproject.org/browse/FLUID-6552) rather than here.

Reply:
@greatislander I think at this point it might be better to keep the contrast theme styles in Themes.styl file that way at least there is an option to build the contrast styles exclusive of the other enactors. Which is a step towards the goals of [FLUID-6552](https://issues.fluidproject.org/browse/FLUID-6552). I get your point about wanting more flexibility with the colour variables. I think eventually we'll also want those in a separate file too, that way you could import them into other styl files or use a completely different set of colour variables if you wanted. Anyways, I think in general not moving the styles out of Themes.styl minimizes the changes in this PR and the broader structural changes can take place in FLUID-6552 or other future JIRAs as needed.

Reply:
Makes sense! I'll move them back.